android/app/src/main/python/mmrelay/plugins/map_plugin.py

```
@@ -0,0 +1,353 @@
+        match = re.match(pattern, full_message)
+
+        # Indicate this message is not meant for this plugin
+        if not match:
+            return False
+
+        zoom = match.group(1)
+        image_size = match.group(2, 3)
+
+        try:
+            zoom = int(zoom)
+        except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)